app/assets/javascripts/group_members.js

```diff
@@ -0,0 +1,33 @@
+$(document).on('opened', '#group-members[data-reveal]', function () {
```

Comment:
This is new javascript to autopopulate the text box when searching for a user to add to a group as a group member.